packages/react-ui-kit/src/Form/Checkbox.jsx

```diff
@@ -40,24 +43,42 @@ const StyledCheckbox = Input.withComponent('input').extend.attrs({
   margin-bottom: 0;
 `;
 
-const Checkbox = ({id, children, ...props}) => {
+const Checkbox = ({id, children, style, ...props}) => {
+  if (id === null) {
```

Comment:
`if (!id) {` ...

Reply:
actually i really want to check against null, as this is the default value i set.

Reply:
oki..

Reply:
but yeah, as every falsy value would not be fit as a key, i am going to change that.